cdc/model/changefeed.go

```diff
@@ -82,20 +82,22 @@ type FeedState string
 
 // All FeedStates
 const (
-	StateNormal   FeedState = "normal"
-	StateError    FeedState = "error"
-	StateFailed   FeedState = "failed"
-	StateStopped  FeedState = "stopped"
-	StateRemoved  FeedState = "removed"
-	StateFinished FeedState = "finished"
+	StateNormal        FeedState = "normal"
+	StatePending       FeedState = "pending"
+	StateFailed        FeedState = "failed"
+	StateStopped       FeedState = "stopped"
+	StateRemoved       FeedState = "removed"
+	StateFinished      FeedState = "finished"
+	StateWarning       FeedState = "warning"
```

Comment:
Please add some comments about each state , such as whether it is visible  to the user or is running in the ticdc

Reply:
ok